### ISSUE TITLE
4.2. Protocol Stack should be Protocol Stack Instance?

### ISSUE THREAD

**Issue by abrunstrom** [taps-api/drafts#89]
In Figure 1 we use Protocol Stack Instance in Section 4.2 we use Protocol Stack. We should be consistent. So one place needs to be updated. Protocol Stack Instance seems right to me.

**Comment by tfpauly** [comment]
At least right now, we use Protocol Stack in the text for the architecture and implementation drafts—adding Instance (while correct) gets a bit wordy in my opinion, and any singular Stack is an instance. I'll update the figure.

**Comment by tfpauly** [comment]
Updated

**Comment by britram** [comment]
this needs to happen in the api doc too

**Comment by britram** [comment]
actually not really; carry on.
